openadapt/utils.py

```diff
@@ -301,7 +436,16 @@ def draw_rectangle(
     return image
 
 
-def get_scale_ratios(action_event):
+def get_scale_ratios(action_event: Any) -> tuple[float, float]:
```

Comment:
Please replace `Any` with `models.ActionEvent`

Reply:
resolved in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/4d079dc4023313ab796a1398cd95506a1885013c